abc_spec/components/schemas/HostedPayments/HostedPaymentsRequest.yaml

```diff
@@ -180,3 +180,14 @@ properties:
         description: Determines whether to attempt a 3D Secure payment as non-3D Secure should the card issuer not be enrolled
         default: false
         example: false
+  capture:
+    type: boolean
+    description: Whether to capture the payment (if applicable)
+    default: true
```

Comment:
I don't think we are/should be setting a default for this one

Reply:
Good point, I had copied these properties from other places in the api reference, but wasn't sure about including the default either - removed now